### PR TITLE
Add `-q` parameter to fourmolu

### PR DESCRIPTION
**Pull request by 8dcc**
For removing lines like `Loaded config from: /home/user/.config/fourmolu.yaml` when formatting.

### CODE REVIEW

**Comment by feature-not-a-bug** [comment]
I appreciate the PR. I can merge this if you could make a change to handle situations where someone overrides the executable with a different formatter.

Rather than hard-coding the value, in `plugin/vim_fourmolu.vim` can you add a check for `g:fourmolu_executable`, and if it's `fourmolu` set a variable called `g:fourmolu_quiet = " -q "`?

**Comment by 8dcc** [comment]
Yeah you are right, I will add it now.

File: plugin/vim_fourmolu.vim [hunk 1]
```diff
@@ -11,6 +11,13 @@ if !exists("g:fourmolu_executable") && executable("fourmolu")
     let g:fourmolu_executable = "fourmolu"
 endif
 
+" If executable contains 'fourmolu' (case insensitve), add -q flag
+if !exists("g:fourmolu_quiet") && g:fourmolu_executable =~? "fourmolu"
+    let g:fourmolu_quiet = " -q "
+else
+    let g:fourmolu_quiet = ""
```
Comment (8dcc) [inline plugin/vim_fourmolu.vim:18]: Not sure if there is a better way of doing this and also checking `!exists("g:fourmolu_quiet")`, but it works.

**Comment by feature-not-a-bug** [comment]
Looks good, thanks for making the change

**Pull request merged**